Screens/PerformanceScreen.js

```diff
@@ -7,15 +7,39 @@ import { connect } from 'react-redux';
 import { loadPerformance } from '../Actions/PerformanceCreator';
 import LocalizeComponent from "../Localization/LocalizedComponent";
 import { BallIndicator } from 'react-native-indicators';
+import ImageLayout from "react-native-image-layout";
+import _ from 'lodash';
+
+
+
+var images = [ // temp images while we don`t have gellery images from site
+    { uri: "https://lvivpuppet.com/wp-content/uploads/2019/01/IMG_3200-300x165.jpg" },
+    { uri: "https://lvivpuppet.com/wp-content/uploads/2019/01/IMG_3196-300x170.jpg", },
+    { uri: "https://lvivpuppet.com/wp-content/uploads/2019/01/IMG_3184-300x169.jpg", },
+    { uri: "https://lvivpuppet.com/wp-content/uploads/2019/01/IMG_3178-300x169.jpg", },
+    { uri: "https://lvivpuppet.com/wp-content/uploads/2019/01/IMG_3165-300x169.jpg", },
+]
 
 class PerformanceScreen extends LocalizeComponent {
 
     componentDidMount() {
         this.props.loadPerformance(this.props.navigation.getParam('performance', 'NO-ID'), this.props.languageCode);
     };
 
+    getCreativeTeamMembers(role) {
```

Comment:
call this in mapStateToProps
Use some role key which is same in all translation 
if you don't have it on backend - create it